module-2/backend/week_four.md

```diff
@@ -10,14 +10,26 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+It's a little chunk of information that is set when a person's computer connects to a site. It is then passed back and forth with the as part of the HTTP request/response.
 * What’s the difference between a session and a cookie?
+A cookie is not encrypted, a session is.
 * What’s a flash and when do you want to use flashes?
+It's a way to display a message on the page that is rendered after the flash is called in the controller. It's a way to pass a message about some action to the user. Like "you were logged in."
 * Why do people say “HTTP is stateless”?
+Because there's no state being passed back and forth from the user to the server. The server just has it's assets.
```

Comment:
Because a stateless protocol does not require the server to retain session information or status about each communications partner for the duration of multiple requests. HTTP is a stateless protocol, which means that the connection between the browser and the server is lost once the transaction ends.